contracts/kleros/KlerosGovernor.sol

```diff
@@ -66,6 +68,7 @@ contract KlerosGovernor is Arbitrable {
 
     uint public submissionDeposit; // Value in wei that needs to be paid in order to submit the list. Note that this value should be higher than arbitration cost.
```

Comment:
Remove `Note that this value should be higher than arbitration cost.` since it is no longer true

Reply:
> since it is no longer true
Is it though? If the submission deposit is low enough there won't be enough eth to fund the dispute.

Reply:
So, my reasoning to use the bytes32 array is that I believed a new memory slot would allow us to make `submissionDeposit` behave [similar to `GTCR.baseDeposit`](https://github.com/kleros/tcr/blob/02b5369f2df7e415b6dc32f152e471539e208f19/contracts/GeneralizedTCR.sol#L625) and avoid it having to update it whenever `arbitrationCosts` goes up.

`submitList()` would like like this:
```
function submitList(address[] _target, uint[] _value, bytes _data, uint[] _dataSize, string _description) public payable duringSubmissionPeriod {
        ...
        require(msg.value >= submissionDeposit + arbitrationCost + sharedStake, "Deposit must be paid in full.");
        ...
```

This means we would change from having part of the deposit paid to submit + a bit more when and if a dispute arises -> requesting the full cost upfront.

Do you think that would require too much code changes?

Reply:
@clesaege what do you think as well?

Reply:
Yeah I understand. I think the problem might be that currently this deposit is not a part of contribute-withdraw logic, like it doesn't belong to any round etc. As of now I don't see anything wrong with it though
So at first glance I think it might be worth changing.

Reply:
Yes, we can do this modification to include arbitration cost as a part of the deposit.